tests/lib.rs

```diff
@@ -154,18 +154,48 @@ fn vcf_baf() {
 }
 
 #[test]
-fn test_oncoprint() {
+fn test_report() {
     assert!(
         Command::new("bash")
             .arg("-c")
-            .arg("target/debug/rbt oncoprint a=tests/test-oncoprint.vcf b=tests/test-oncoprint.vcf > tests/oncoprint.html")
+            .arg("target/debug/rbt vcf-report tests/ref.fa a=tests/report-test.vcf a=tests/test-report.bam b=tests/report-test.vcf b=tests/test-report.bam -- tests")
```

Comment:
I would prefer the following CLI:
```
rbt vcf-report tests/ref.fa --vcfs a=tests/report-test.vcf b=tests/report-test.vcf --bams a=tests/test-report.bam  b=tests/test-report.bam -- tests
```

Reply:
Changed the CLI with https://github.com/rust-bio/rust-bio-tools/pull/53/commits/240770d4e7ae2056b49fa168576a90d6c972656d according to the specification.